Subsonicious/Subsonicious/Views/LoginView.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  LoginView.swift
+//  Subsonicious
+//
+//  Created by Bilal on 15/06/2020.
+//  Copyright © 2020 Bilal Benlarbi. All rights reserved.
+//
+
+import SwiftUI
+
+struct LoginView: View {
+
+    @EnvironmentObject var authentication: Authentication
+    @State var server = Server()
+
+    var body: some View {
+        GeometryReader { content in
+            VStack {
+                Text("login.login")
+
+                VStack {
+                    TextField("login.server", text: self.$server.address)
+                        .keyboardType(.URL)
+                        .textContentType(.URL)
+                        .asLoginField()
+
+                    TextField("login.username", text: self.$server.username)
+                        .keyboardType(.asciiCapable)
+                        .textContentType(.username)
+                        .asLoginField()
+
+                    SecureField("login.password", text: self.$server.password)
+                        .textContentType(.password)
+                        .asLoginField()
+                }
+
+                Button(action: {
+                    self.continueButtonPressed()
+                }) {
+                    Text("login.continue")
+                        .font(.headline)
+                        .foregroundColor(.white)
+                        .frame(maxWidth: content.size.width, maxHeight: 44)
+                        .background(Color.yellow)
+                        .cornerRadius(6)
+                        .opacity(self.continueButtonOpacity)
+                }
+                .disabled(self.shouldDisableContinueButton)
```

Comment:
I think you can rename the variable to `continueButtonIsDisabled` to symbolise the current state, not a futur with should.